src/stylesheets/layouts/_header.scss

```diff
@@ -2,44 +2,50 @@
   background-color: $color-mint;
   display: flex;
   justify-content: center;
+  position: relative;
 
   &__wrapper {
-    margin: 50px;
-    // position: relative;
+    max-width: 50rem;
+    margin: 5rem 6rem;
   }
-  // &__primary {
-  //   display: flex;
-  //   justify-content: center;
-  //
-  //   &--top {
-  //     display: block;
-  //     margin-left: auto;
-  //     margin-right: auto;
-  //     text-align: center;
-  //   }
-  //
-  //   &--btm {
-  //     display: block;
-  //     width: 500px;
-  //   }
-  // }
-  &__text-box {
+
+  &__primary {
+    width: 100%;
```

Comment:
since this is an `<h1>`, it should be `display: block` and `width: 100%` by default - so you may not need these two properties.

Reply:
Most elements are "block-level" elements and therefore `display: block` and `width: 100%` by default.

The exceptions which are "inline" elements by default would be `<span>`(s), `<a>`(s), `<strong>`, `<em>`, etc